### ISSUE TITLE
Enum literals not inserted using Compile on Save, but works with project build.

### ISSUE THREAD

**Issue by mrcrowl** [Microsoft/TypeScript#1916]
Since TypeScript 1.4, using the Compile on Save feature, enum members are not being replaced with their literal values in the output js file.

The same operation via a full project build inserts the literal value as expected.  This occurs for both regular and const enums.

In the linked example project, there is an enum file "Animals.ts" which is being AMD-imported into the class file "Farm.ts".  If you make a minor edit to the Farm.ts file (e.g. insert a space and save), then line 4 of Farm.js shows:
```
            this.animals = [Animals.Chicken, Animals.Cow, Animals.Sheep];
```
Whereas, if you do a project build, the line 4 of Farm.js shows the expected output:
```
            this.animals = [3 /* Chicken */, 0 /* Cow */, 1 /* Sheep */];
```

Example project: https://www.dropbox.com/s/xc34f0q6rhc0amm/ExampleProject.zip?dl=0

I can confirm that this is happening in both VS2013 + TS1.4 and VS2015 CTP.

**Comment by vladima** [comment]
looks like a duplicate of #1742, can you try to build the latest bits from the master and check if this issue can still be reproduced?

**Comment by mrcrowl** [comment]
@vladima Yeah sorry, I found the other issues after I had posted.

I've built from the master and substituted in the resulting typescriptServices.js, but then I get this error when I try and open any ts file in VS2013:

API version mismatch: managed version '0.4', script version '0.5'

**Comment by vladima** [comment]
You can try to rollback to the revision prior to this commit (394d6f4f0e58a037da0e1af2181d55aaaab31945) that has bumped the version - it should already contain the fix of the enum issue

**Comment by mrcrowl** [comment]
Thanks @vladima.  For anyone else wanting the correct typescriptServices.js file (pre-compiled), you can find it here: https://dl.dropboxusercontent.com/u/14232632/typescriptServices.js

**Comment by vladima** [comment]
@mrcrowl do you still see the issue with updated typescriptServices.js ?

**Comment by NoelAbrahams** [comment]
@vladima, the issue appears to have been fixed in the updated typeScriptServices.js

**Comment by vladima** [comment]
great to hear that!

**Comment by mrcrowl** [comment]
@vladima Yes, this was fixed for me by using 26929ee5f28c7883698f322ecb4379c69c6a6b0e.
